### ISSUE TITLE
is_blacklisted was never awaited

### ISSUE THREAD

**Issue by archrinto** [NovemberOscar/Sanic-JWT-Extended#22]
I got some error when i add my own blacklist class
`RuntimeWarning: coroutine 'TokenBlacklist.is_blacklisted' was never awaited`

Anyone can help?

thanks.

**Comment by rscarrera27** [comment]
Hello! 

Basically, it is just asyncio warning so that never effect on your program. 

But if you give me a source code fragment or repo link, I think I can help you a little

**Comment by rscarrera27** [comment]
\+ Traceback also helpful

**Comment by archrinto** [comment]
this is my class,

`class TokenBlacklist(BlacklistABC):
    async def register(self, token: Token):
        blacklist = AuthBlacklist(access_token=token.raw_data['jti'])
        await blacklist.commit()

    async def is_blacklisted(self, token: Token):
        blacklist = await AuthBlacklist.find_one({'_id': token.raw_data['jti']})
        return blacklist != None`

when i access this method with valid token, i got that error and then got "Token has been revoked" message.

`@app.route('/users/<id>', methods=['GET'])
@jwt_required
async def get_user(req, id, token):
        ...
`

**Comment by archrinto** [comment]
this is my class
```
class TokenBlacklist(BlacklistABC):
    async def register(self, token: Token):
        blacklist = AuthBlacklist(access_token=token.raw_data['jti'])
        await blacklist.commit()
        
    async def is_blacklisted(self, token: Token):
        blacklist = await AuthBlacklist.find_one({'_id': token.raw_data['jti']})
        return blacklist != None
```
when i access this method with valid token, i got that error and then got "Token has been revoked" message.
```
@app.route('/users/<id>', methods=['GET'])
@jwt_required
async def get_user(req, id, token):
     ...
```
this is the error, in terminal
```
python3.6/site-packages/sanic_jwt_extended/decorators.py:165: 
RuntimeWarning: coroutine 'TokenBlacklist.is_blacklisted' was never awaited
  if JWT.config.use_blacklist and JWT.blacklist.is_blacklisted(token_obj):
```

**Comment by rscarrera27** [comment]
OK I found why.` is_blacklisted` is async method. 
 but `sanic_jwt_extended/decorators.py:165` is never await it.

```python
if JWT.config.use_blacklist and JWT.blacklist.is_blacklisted(token_obj):
```

and not awaited coro object will always be considered as true. so even token is valid you got revoked warning. 

This is my mistake. I will fix it now.

Thank you.

**Comment by rscarrera27** [comment]
@archrinto patched `1.0.dev10` version released. plz update :>

**Comment by archrinto** [comment]
Thanks @NovemberOscar
